app/models/micropost.rb

```diff
@@ -0,0 +1,15 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope ->{order(created_at: :desc)}
```

Comment:
trả lời câu hỏi: nhược điểm của default_scope là gì?

Reply:
Nhược điểm của default_scope: - Ảnh hưởng tới toàn bộ câu truy vấn khi lấy dữ liệu, nên gây khó chịu trong những trường hợp mình không muốn dữ liệu lấy về không bị tác động của default_scope.
Ví dụ: Đối tượng "Bài đăng" có thuộc tính activated, mình dùng default_scope -> {where(activated: true)} để hiện thị những bản ghi có activated = true. (Dùng để hiện thị bên trang public) . Nhưng khi ta cũng muốn hiện thị những bản ghi của "Bài đăng" ở phần admin, nếu mình không để ý thì dữ liệu chỉ trả về những bản ghi có activated = true.